projects/wx31.cmake

```diff
@@ -1,7 +1,7 @@
 # wx31
 xpProOption(wx31 DBG_MSVC)
 set(REPO github.com/wxWidgets/wxWidgets)
-set(FORK github.com/smanders/wxWidgets)
+set(FORK github.com/cameronfrandsen/wxWidgets)
```

Comment:
for future reference: you shouldn't need to change this... certainly never need to commit it

the only reason I can see committing this is to verify that the README.md patch/diff link shows your commits, but that's really not necessary

you can navigate to the `xpbase/Source/wx31_repo` and add a remote to your fork, set the branch or commit or local changes to what you wish to be in the patch, then go back to the build directory where you ran `XP_STEP=mkpatch` and build -- the patch file in the externpro `patches/` directory should show the changes you want to commit and submit

let me know if you'd like any clarification on this